packages/just-scripts/src/monorepo/getStackDiffs.ts

```diff
@@ -0,0 +1,47 @@
+import fs from 'fs-extra';
+import glob from 'glob';
+import { downloadPackage, exec, paths, encodeArgs } from 'just-scripts-utils';
+import path from 'path';
+import { DiffInfo } from './DiffInfo';
+import { diff_match_patch, patch_obj, Diff } from 'diff-match-patch';
+
+export async function getStackDiffs(stack: string, fromVersion: string, toVersion: string): Promise<DiffInfo> {
+  const packagePaths = await Promise.all([downloadPackage(stack, fromVersion), downloadPackage(stack, toVersion)]);
+
+  // Concentrates on new and modified files only
+  const diffInfo: DiffInfo = {
+    patches: {},
+    fromVersion,
+    toVersion
+  };
+
+  const fromPath = packagePaths[0]!;
+  const toPath = packagePaths[1]!;
+  const globbedFiles = glob.sync('**/*', { cwd: toPath, nodir: true }).concat(glob.sync('**/.*', { cwd: toPath, nodir: true }));
```

Comment:
Should be able to do this in one glob with `dot: true`

Reply:
oh good call! didn't know :)